### PR TITLE
added error messages to api call errors

### PR DESCRIPTION
**Pull request by hinriksteinar**
# Error messages to api errors

Attach a link to issue if relevant

## Screenshots / Gifs

Attach Screenshots / Gifs to help reviewers understand the scope of the pull request

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

### CODE REVIEW

File: libs/application/templates/document-provider-onboarding/src/fields/TestPhase/AutomatedTests/index.tsx [hunk 1]
```diff
@@ -27,6 +27,7 @@ const AutomatedTests: FC<FieldBaseProps> = () => {
 
   const [response, setResponse] = useState<Response[]>([])
   const [isLoading, setIsLoading] = useState(false)
+  const [automatedTestsError, setautomatedTestsError] = useState('')
```
Comment (fjandakornid) [inline libs/application/templates/document-provider-onboarding/src/fields/TestPhase/AutomatedTests/index.tsx:30]: myndi nota useState<string | null>(null)

File: libs/application/templates/document-provider-onboarding/src/fields/TestPhase/AutomatedTests/index.tsx [hunk 2]
```diff
@@ -40,7 +41,8 @@ const AutomatedTests: FC<FieldBaseProps> = () => {
     })
```
Comment (fjandakornid) [inline libs/application/templates/document-provider-onboarding/src/fields/TestPhase/AutomatedTests/index.tsx:41]: setja setautomatedTestError sem null í byrjun á validateEndpoint til að clear-a errors ef hann reynir aftur, því ef þetta tekst í skipti nr 2 þá myndi villan halda sér.

File: libs/application/templates/document-provider-onboarding/src/fields/TestPhase/AutomatedTests/index.tsx (hunk 1 above)
Comment (fjandakornid) [reply libs/application/templates/document-provider-onboarding/src/fields/TestPhase/AutomatedTests/index.tsx:30]: og breyta seinni gaurnum í setAutomated... þ.e. stórt A

Comment (hinriksteinar) [reply libs/application/templates/document-provider-onboarding/src/fields/TestPhase/AutomatedTests/index.tsx]: Búinn að breyta useState

File: libs/application/templates/document-provider-onboarding/src/fields/TestPhase/AutomatedTests/index.tsx (hunk 2 above)
Comment (hinriksteinar) [reply libs/application/templates/document-provider-onboarding/src/fields/TestPhase/AutomatedTests/index.tsx:44]: Bætti við null reset í byrjun.

**Review by ssh16 — APPROVED** [review]

**Pull request merged**